ironfish/src/blockchain/blockchain.ts

```diff
@@ -958,11 +964,15 @@ export class Blockchain {
 
       graffiti = graffiti ? graffiti : Buffer.alloc(32)
 
+      const transactionHashes = transactions.map((t) => t.hash())
+      const transactionCommitment = transactionMerkleRoot(transactionHashes)
```

Comment:
We'll want to check this commitment in a Verifier function at some point, so we can detect if the transactions don't match the block header commitment.

Reply:
Added to `verifyBlock`